source/particle/property/interface.cc

```diff
@@ -286,6 +286,56 @@ namespace aspect
 
 
 
+      template <int dim>
+      void
+      IntegratorProperties<dim>::initialize_one_particle_property(const Point<dim> &/*position*/,
+                                                                  std::vector<double> &data) const
+      {
+        data.resize(data.size() + n_integrator_properties);
+      }
+
+
+
+      template <int dim>
+      std::vector<std::pair<std::string, unsigned int> >
+      IntegratorProperties<dim>::get_property_information() const
+      {
+        const std::vector<std::pair<std::string,unsigned int> > property_information (1,
+                                                                                      std::make_pair("integrator properties",n_integrator_properties));
+        return property_information;
```

Comment:
Or just directly return that curly-braced enclosed list here.

Reply:
Ping?

Reply:
I find returning curly-braced lists hard to ready for non-c++ experts, but I changed it now. Initializer lists are weird for everyone coming from a c-background (not saying we should not use them, I just try to avoid using them when not necessary).